.github/workflows/testing.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
Do we still test on 3.10? I thought we have 3.11 as the minimum of ert/everest?

Reply:
Good point